src/Microsoft.Identity.Client/ApiConfig/Executors/PublicClientExecutor.cs

```diff
@@ -136,7 +136,10 @@ public PublicClientExecutor(IServiceBundle serviceBundle, PublicClientApplicatio
             coreUiParent.UseCorporateNetwork = _publicClientApplication.AppConfig.UseCorporateNetwork;
 #endif
 #endif
-            return ServiceBundle.PlatformProxy.GetWebUiFactory().CreateAuthenticationDialog(coreUiParent, requestContext);
```

Comment:
>requestContext [](start = 106, length = 14)

shouldn't the additional params be available via the context? e.g. requestContext could probably hold a reference to either service bundle or the platformproxy, and perhaps even to the coreUIParent ?